build.gradle

```diff
@@ -141,6 +141,7 @@ subprojects {
     def buildLog = new File(logFile)
     addTestOutputListener(new TestOutputListener() {
       def lastDescriptor
+
```

Comment:
Nit:  please don't introduce any unrelated changes for avoiding conflicts when rebase to people's own repo.

Reply:
sorry. that was a mistake. will fix